home-server-stub/pom.xml

```diff
@@ -24,6 +24,10 @@
             <version>${revision}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jersey</artifactId>
+        </dependency>
     </dependencies>
```

Comment:
Please rename the current module&artifactId to `home-server-generator`.

Reply:
Ok, noted